esf_main.tex

```diff
@@ -271,12 +271,11 @@ \subsubsection{Description (type, operation parameters)}
 	\end{table}
 
 \subsubsection{Wiring/cables/connectors/}
-%Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.
-TODO
+            The Inertia switch will be wired to be normally closed and open the shutdown circuit in the case that there is a crash. The inertia switch is wired in-line with the shutdown circuit to be normally closed. Please see \fref{fig:shutdown_circuit} for the position of the inertia switch relative to the other shutdown system components.
```

Comment:
This feels a bit wordy/repetitive. Possibly remove the part in the first sentence about the Inertia Switch being normally closed, and just leave that for the second sentence.